.golangci.yml

```diff
@@ -28,24 +28,15 @@ linters-settings:
 
   # https://github.com/ultraware/funlen
   funlen:
-    lines: 100
-    statements: 50
+    lines: 140
+    statements: 70
+    exclude:
+      - "^\/\/"
```

Comment:
does this account for spaces in front `//`?

Reply:
update: funlen doesn't have an exclude setting